src/main/xsl/preprocess/maprefImpl.xsl

```diff
@@ -389,6 +394,23 @@ See the accompanying LICENSE file for applicable license.
     </xsl:call-template>
   </xsl:template>
 
+  <xsl:template match="@*" mode="preserve-submap-attributes">
+    <xsl:attribute name="dita-ot:submap-{local-name()}" select="."/>
+  </xsl:template>
+  
+  <xsl:template match="*[contains(@class,' topic/title ')]" mode="preserve-submap-title-and-topicmeta">
+    <dita-ot:submap-title class="+ topic/foreign ditaot-d/submap-title ">
```

Comment:
Yes, I meant we wrap navtitle into topicmeta, per the spec. But if we get an error about topicgroup having a title, that would need to be adjusted too.